### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by HarryEntorno**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <title> Welcome to Potionfy </title>
 </head>
 <body>
-  
+	<h1> welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (hermioneentorno) [inline index.html:10]: Could you remove the Enjoy your stay. string and make the Welcome to Potionfy! string red?

Comment (hermioneentorno) [inline index.html]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

**Review by hermioneentorno — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (HarryEntorno) [reply index.html]: Sure, can do!

**Review by hermioneentorno — CHANGES_REQUESTED** [review]
Please change the code as per my comments on the review

**Review by hermioneentorno — APPROVED** [review]

**Pull request merged**